.eslintrc.json

```diff
@@ -52,7 +49,9 @@
       }
     ],
     "sort-vars": "error",
-    "strict": ["error", "global"]
+    "strict": ["error", "global"],
+    "react-hooks/rules-of-hooks": "error",
+    "react-hooks/exhaustive-deps": "warn"
```

Comment:
Can we set this to `"error"` as well? Seems like we would want to block merging for these rules.